understanding/21/target-size-enhanced.html

```diff
@@ -7,87 +7,103 @@
-			<p>The issue can be further complicated for responsive/mobile sites which need to accommodate different types of fine and coarse inputs (e.g. a site that can be accessed both on a traditional desktop/laptop with a mouse, as well as on a tablet or mobile phone with a touch screen).</p>
-			<p>While this criterion defines a minimum target size, it is recommended that larger sizes are used to reduce the possibility of unintentional actions. This is particularly relevant if any of the following are true:</p>
-	  <ul>
-			  <li>the control is used frequently;</li>
-			  <li>the result of the interaction cannot be easily undone;</li>
-			  <li>the control is positioned where it will be difficult to reach, or is near the edge of the screen;</li>
-			  <li>the control is part of a sequential task.</li>
-	  </ul>
-		<section id="benefits">
-			<h3>Benefits</h3>
-			<ul>
-			  <li>Users who use a mobile device where touch screen is the primary mode of interaction</li>
-			  <li>Users with mobility impairments, such as hand tremors</li>
-			  <li>Users who find fine motor movements difficult</li>
-			  <li>Users who access a device using one hand</li>
-			  <li>Users with large fingers, or who are operating the device with only a part of their finger or knuckle</li>
-			  <li>Users who have low vision may better see the target</li>
-		  </ul>
-		</section>
+  	<section id="brief">
+  		<h2>In brief</h2>
+  		<dl>
+  			<dt>Goal</dt><dd>Controls can be operated more easily, especially on touch screens.</dd>
+  			<dt>What to do</dt><dd>Make custom targets at least 44 by 44 pixels.</dd>
+  			<dt>Why it's important</dt><dd>Some people cannot tap small objects.</dd>
+  		</dl>
+  	</section>
+  	<section id="intent">
+  		<h2>Intent</h2>
+  		<p>The intent of this success criterion is to help users who may have trouble activating a small target because of hand tremors, limited dexterity or other reasons. If the target is too small, it may be difficult to aim at the target. Mice and similar pointing devices can be hard to use for these users, and a larger target will help them greatly in having positive outcomes on the web page.</p>
+      <p>Touch is particularly problematic as it is an input mechanism with coarse precision. Users lack the same level of fine control as on inputs such as a mouse or stylus. A finger is larger than a mouse pointer, and generally obstructs the user's view of the precise location on the screen that is being touched/activated.</p>
+      <p>The issue can be further complicated for responsive/mobile sites which need to accommodate different types of fine and coarse inputs (e.g. a site that can be accessed both on a traditional desktop/laptop with a mouse, as well as on a tablet or mobile phone with a touch screen).</p>
```

Comment:
Line 22: 
1. Replace the slashes with the word "or" to improve readability
2. Replace "which" with "that" (since it's followed by a restrictive clause)
3. Replace "e.g." with "for example" for plainer language
4. Consider rewriting this as two sentences to decrease the cognitive load

As one sentence:
"The issue can be further complicated for responsive and mobile sites that need to accommodate different types of fine and coarse inputs (for example, a site that can be accessed on a traditional desktop or laptop with a mouse, as well as on a tablet or mobile phone with a touch screen)."

Reply:
Thanks, @frankie-wolf. 
The purpose of this PR was just to restore accidentally deleted material, not to alter that content.
There have been a number of suggestions by people of ways to improve the content, and our intention is to create a second PR which improves the information, not just restores it!